nservicebus/operations/auditing.md

```diff
@@ -77,6 +77,8 @@ snippet:AuditWithCode
 
 ### Using app.config
 
+Warning: Configuration of auditing via `app.config` is not recommended in Version 6, use the [code configuration API](#configuring-auditing-using-code) instead. The configuration section will be removed in Version 7.
```

Comment:
i know currently the warnings are specific to each use case. but can we come up with a generic one and put it in a `.include.md`?

eg

> Warning: Configuration this feature via `app.config` is not recommended in Version 6,  instead use the code based API listed above. The configuration section will be removed in Version 7.